changelogs/fragments/T7002-firewall-global-integration-tests.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - fix integration tests for `firewall_global` for v1.3-
```

Comment:
Update to include a summary of the actual fixes as well. Since this wasn't just changes to the integration tests, but the code that isn't working with 1.3- correctly.